project/MiMa.scala

```diff
@@ -889,7 +889,11 @@ object MiMa extends AutoPlugin {
         ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.scaladsl.model.HttpMessage.discardEntityBytes"),
 
         // #20288 migrate BodyPartRenderer to GraphStage
-        ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.impl.engine.rendering.BodyPartRenderer.streamed")
+        ProblemFilters.exclude[IncompatibleResultTypeProblem]("akka.http.impl.engine.rendering.BodyPartRenderer.streamed"),
+
+        // #21727 moved all of Unfold.scala in package akka.stream.impl
+        ProblemFilters.exclude[MissingClassProblem]("akka.stream.scaladsl.UnfoldAsync"),
+        ProblemFilters.exclude[MissingClassProblem]("akka.stream.scaladsl.Unfold")
```

Comment:
It should go to `2.4.11` as it is the current released version of akka when this binary incompatible change is introduced.